src/System.Private.CoreLib/shared/System/ComponentModel/EditorBrowsableAttribute.cs

```diff
@@ -7,40 +7,31 @@ namespace System.ComponentModel
     [AttributeUsage(AttributeTargets.Class | AttributeTargets.Struct | AttributeTargets.Enum | AttributeTargets.Constructor | AttributeTargets.Method | AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Event | AttributeTargets.Delegate | AttributeTargets.Interface)]
     public sealed class EditorBrowsableAttribute : Attribute
     {
-        private EditorBrowsableState browsableState;
-
         public EditorBrowsableAttribute(EditorBrowsableState state)
         {
-            browsableState = state;
+            State = state;
         }
 
-        public EditorBrowsableAttribute() : this(EditorBrowsableState.Always) { }
-
-        public EditorBrowsableState State
+        public EditorBrowsableAttribute() : this(EditorBrowsableState.Always)
         {
-            get { return browsableState; }
         }
 
+        public EditorBrowsableState State { get; }
+
         public override bool Equals(object obj)
         {
             if (obj == this)
             {
                 return true;
             }
+            if (!(obj is EditorBrowsableAttribute other))
+            {
+                return false;
+            }
 
-            return (obj is EditorBrowsableAttribute other) && other.browsableState == browsableState;
-        }
-
-        public override int GetHashCode()
-        {
-            return base.GetHashCode();
+            return other.State == State;
```

Comment:
Why did you change this from what it was?  That's a very common form for writing out such statements.